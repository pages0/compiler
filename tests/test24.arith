( 1+ 5*8 -3, 4>3 and (5-3) <3)