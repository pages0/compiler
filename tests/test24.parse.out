( pair ( - ( + 1 ( * 5 8 ) ) 3 ) ( and ( > 4 3 ) ( < ( - 5 3 ) 3 ) ) )